group/urls.py

```diff
@@ -6,6 +6,7 @@
     path('', GroupApiView.as_view()),
     path('<int:group_id>/', GroupApiView.as_view()),
     path('detail/<int:group_id>/', GroupDetailApiView.as_view()),
-    path('user/<int:user_id>/', GroupDetailApiView.as_view()),
-    path('message', MessageAPIView.as_view()),
+    path('users/<int:user_id>/', GroupDetailApiView.as_view()),
+    path('messages', MessageAPIView.as_view()),
+    path('messages/<int:user_id>/', MessageAPIView.as_view()),
```

Comment:
Why are we using the same class on this URL as the one above it?

Reply:
If you are referring to the MessageAPIView class then it only makes sense for both the POST and GET methods to be within the same class hence the URLs